### ISSUE TITLE
Problem with 'secret' parameter in parameters.yml.dist

### ISSUE THREAD

**Issue by migmolrod** [Sylius/Sylius#9439]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | (not sure, read below)
| RFC?             | no
| Sylius version   | 1.1.6

Secret parameter is missing a % at the end. Right now it is:
```yaml
parameters:
    secret: "%env(SYLIUS_SECRET)"
```
It should be:
```yaml
parameters:
    secret: "%env(SYLIUS_SECRET)%"
```
Right now, every single sylius project is using exactly the same secret by default: `%env(SYLIUS_SECRET)` (taken as a literal, it's not parsed because the lack of % at the end) even if the developers change the `env(SYLIUS_SECRET)` parameter itself.

BTW, could this change 'break' existing databases? Is the secret used while hashing admin_user and shop_user passwords and if we change this, symfony won't be able to authenticate existing users created with the old `secret` parameter?

**Comment by pamil** [comment]
It is not a BC break, secret is used for remember me functionality and CSRF tokens, password hashes use salt which is generated randomly for every user.

**Comment by pamil** [comment]
Fixed by #9440, thanks for reporting @migmolrod! :)
